### ISSUE TITLE
Config parser should reject unknown keys

### ISSUE THREAD

**Issue by Minoru** [newsboat/newsboat#568]
**Newsboat version (copy the output of `newsboat -v` or the first line of `git show`)**: 2.15

Steps to reproduce the issue:

1. Put this into the config (copied from #567):

```
unbind-key wheel-up
unbind-key wheel-down
```

2. Run Newsboat.

Newsboat starts just fine, even though "wheel-up" and "wheel-down" don't exist and aren't documented.

Expected behaviour is for Newsboat to *not* start, and print an error message instead.

**Comment by allanwind** [comment]
I added `run-on-startup end; open` to my config and newsboat 2.13.0 fails with `Loading configuration...unknown command `run-on-startup'` but it's a valid command in r2.24-70-g41f1b.  This is the behavior you are asking for in the above, however that breaks forward of the config file.  I believe forward compatibility was important to you so the desired behavior actually should be a warning and an entry int he error log if enabled but start irregardless, no?

**Comment by Minoru** [comment]
> This is the behavior you are asking for in the above

No, I'm asking to check the names of the keys which are passed into `unbind-key`.

Forward compatibility is important to me with regards to cache.db and the queue file, because these files are managed by Newsboat itself. On the other hand, the config (and the urls file) is totally under user's control, so we should have diagnostics that reject invalid or broken configs.

**Comment by allanwind** [comment]
Thanks for the clarification.
